app/views/pages/index.html.haml

```diff
@@ -0,0 +1,122 @@
+%main.container
+    #slider.carousel.slide{'data-ride': "carousel"}
+        %ol.carousel-indicators
+            %li.active{'data-target': "#slider", 'data-slide-to': "0"}
+            %li{'data-target': "#slider", 'data-slide-to': "1"}
+            %li{'data-target': "#slider", 'data-slide-to': "1"}
+        .carousel-inner{role: "listbox"}
+            .item.active
+                .row
+                    .col-sm-5
+                        .general-img-wrap
+                            %img.img-shadow.general-thumbnail-img{src: "http://jonathantweedy.com/resources/thumbs/SmashingBook5ResponsiveWebDesign.jpg", alt: "design-book"}
+                    .col-sm-7
+                        .mb-60
+                            %h1 Real-Life Responsive Web Design
+                            %p.lead Smart front-end techniques and design patterns derived from real-life responsive projects
+                            %p.lead.small Vitaly Friedman, Daniel Mall,Vitaly Friedman
+                            .general-align
+                                %button.disabled.btn= I18n.t('home_page.btn_buy')
```

Comment:
Use just `t` helper
```suggestion
                                %button.disabled.btn= t('home_page.btn_buy')
```

Reply:
Also, use lazy lookup for I18n

https://guides.rubyonrails.org/i18n.html#lazy-lookup